profiles/base.yaml

```diff
@@ -845,3 +845,9 @@ force_healer_town:
 shard_thief_password:
 forge_container:
 spell_cycle:
+
+restock_shop_inside_room:
+restock_shop_outside_room:
+restock_shop_entrance_noun: entrance
```

Comment:
❓ Is this (and line 852) a 'common enough' value to use as a default?

Reply:
No, ill blank that out

Reply:
I think Crossing is a reasonable default

Reply:
To be clear the town setting determines where you buy parts from

Reply:
👍

Reply:
Removing the town setting, `hometown` works well enough